tests/test_health.py

```diff
@@ -796,3 +796,25 @@ def test_positive_check_foreman_proxy_verify_dhcp_config_syntax(ansible_module):
 
     :CaseImportance: Medium
     """
+
+
+def test_positive_remove_job_file(setup_subscribe_to_cdn_dogfood, ansible_module):
+    """Verify file /var/lib/pulp/job1.0.0 is not present after the following command.
+
+    :id: eed224f9-a2ec-4d15-9047-cede0b823866
+
+    :setup:
+        1. foreman-maintain should be installed.
+
+    :steps:
+        1. Run foreman-maintain health check --label disk-performance
+
+    :expectedresults: /var/lib/pulp/job1.0.0 file should not exist.
+
+    :CaseImportance: Medium
+
+    :BZ: 1762302
+    """
+    contacted = ansible_module.command(Health.check({"label": "disk" "-performance"}))
+    contacted = ansible_module.find(paths="/var/lib/pulp", file_type="file")
+    assert "job" not in contacted.values()[0]["files"][0]["path"]
```

Comment:
Hello Akhil, I tried to run the test against one instance where `/var/lib/pulp` was populated with some real content. It seems that it's possible to have even ordinary files in that directory, which causes the test to pass even in case of failure (when you roll-back the fix or create a `job` file manually), because only the first file (`["files"][0]`)is checked by the assert.

I would propose to replace the assert with something like this:
```
    for file in contacted.values()[0]["files"]:
        assert "job" not in file["path"]
```
Thank you.

Reply:
Done. :)